docs/styling.md

```diff
@@ -3,126 +3,193 @@ id: styling
 title: Styling
 ---
 
-## Adding styles
-As every component uses Shadow DOM by default, you have to put the styles inside the shadow root.
-You can either import your css from an external file (recommended way for bio-element). In this case
-your imported style has to be a string with the css inside:
+Web components use a shadow DOM to write their HTML. The shadow DOM has some interesting features
+that make it very different from writing to an element using `innerHTML`. Styling-wise, the shadow
+DOM scopes the style, so that styles from the page does not leak into the component and style and
+vice-versa.
 
-```javascript
-import Component from '@biotope/element';
-import style from './styles.css';
-
-class MyButton extends Component {
-  render() {
-    return this.html`
-      Hello World 🐤
-      ${this.createStyle(style)}
-    `;
-  }
-}
+However, applying styles to any HTMLElement is just like applying styles to any `biotope-element`
+component. You basically have two approaches: the `style` element and inline styling - with the same
+advantages and disadvantages as normal HTMLElements.
+
+> __⚠️ Important:__ When adding styles to IE11 specifically, the browser will sometimes not fully
+render the style you applied to your component even though your CSS is fully loaded onto the page.
+This is easily solved by moving your CSS to the bottom of the component (like in all our examples).
+
+> __⚠️ Important:__ Style-scoping only works in browsers that require the Web Component polyfill.
+Writing styles for these components is like writing styles for the entire page. So use unique class
+names (for example [BEM](http://getbem.com)) or a build system that renames your classes to keep
+your codebase maintainable and sane.
 
-MyButton.componentName = 'my-button';
+> __⚠️ Important:__ Class names that are equal to your component name should also be avoided as the
+Web Components polyfill also creates this class as part of the polyfilling process.
 
-MyButton.register();
+## Style element
+Writing a `style` element to a `biotope-element` component is just as simple as writing another
+`div`, like we've shown you before. Here's a simple example:
+
+```javascript
+// render function
+return this.html`
+  <div>
+    Hello World 🐤
+  </div>
+  <style>
+    color: red;
+  </style>
+`;
 ```
 
-or use inline styles:
+### createStyle function
+We provide you with a pretty cool function to do most of this automatically - the `createStyle`
+function. It can take either take a string (or an object with a `toString` function) and convert it
```

Comment:
> It can take either take a string

Should have been:

> It can either take a string

But... I was expecting an `or` and didn't find it. Like:

> It can either be A or B.